### PR TITLE
Add pyilluminate

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyilluminate/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "pyilluminate" %}
```
Comment (chrisburr) [inline recipes/pyilluminate/meta.yaml:1]:
```suggestion
{% set name = "pyIlluminate" %}
```

File: recipes/pyilluminate/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # I can't seem to download from pypi....
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # sha256: c94721144ee4388175edaa31f14388da3101b4583a3368f047fdcf7797da1a4d
+  url: https://github.com/ramonaoptics/pyilluminate/archive/{{ version }}.tar.gz
+  sha256: 7e30a85aa1c3d49e6f59f053f2f969cc77b2505c764dcf244d52bd65878195f8
```
Comment (chrisburr) [inline recipes/pyilluminate/meta.yaml:13]: This should work when the name is changed:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: c94721144ee4388175edaa31f14388da3101b4583a3368f047fdcf7797da1a4d
```

Comment (hmaarrfk) [reply recipes/pyilluminate/meta.yaml:13]: Whoa, thanks, I'm just going to change the name to lowercase on pypi ;). Lets see how badly I'm going to break things.

**Comment by hmaarrfk** [comment]
Thanks!

**Pull request merged**